### ISSUE TITLE
Support string descriptors for `generate`

### ISSUE THREAD

**Issue by andykenward** [apollographql/apollo-codegen#331]
When running 
```
apollo-codegen generate **/*.graphql --schema schema.json --target typescript --addTypename --output operation-result-types.ts
```
If there are string descriptors in a `.graphql` files

`"this is my comment"`

Then an error is thrown of 
```
Unexpected String "this is my comment"
```

See
 - https://github.com/Swydo/meteor-graphql/issues/12#issuecomment-352675055

- https://github.com/graphql/graphql-js/pull/927

**Comment by andykenward** [comment]
Even with having `"graphql": "^0.12.3",` as a dependency in my project it was still using an older version.

I've managed to get it to use a newer version of `graphql`.

Using `yarn ^1.3.2` and adding to my projects `package.json`
```
  "resolutions": {
    "graphql": "^0.12.3"
  }
```

**Comment by martijnwalraven** [comment]
This should be fixed in `apollo-codegen` 0.18.3.
